pandas/core/series.py

```diff
@@ -1002,24 +1002,34 @@ def _set_value(self, label, value, takeable=False):
 
     def reset_index(self, level=None, drop=False, name=None, inplace=False):
         """
-        Analogous to the :meth:`pandas.DataFrame.reset_index` function, see
-        docstring there.
+        Reset the index of the Serie.
+        
+        For an Index, the index name will be used (if set), 
+        otherwise a default index or `level_0` (if `index` is already taken) 
+        will be used.
+        For a MultiIndex, return a new Series with labeling
+        information in the columns under the index names, defaulting to
+        `level_0`, `level_1`, etc. if any are None. 
 
         Parameters
         ----------
-        level : int, str, tuple, or list, default None
+        level : int, str, tuple, or list, default `None`
             Only remove the given levels from the index. Removes all levels by
-            default
-        drop : boolean, default False
-            Do not try to insert index into dataframe columns
-        name : object, default None
-            The name of the column corresponding to the Series values
-        inplace : boolean, default False
-            Modify the Series in place (do not create a new object)
+            default.
+        drop : `bool`, default `False`
+            Do not try to insert index into dataframe columns.
+        name : `object`, default `None`
+            The name of the column corresponding to the Series values.
+        inplace : `bool`, default `False`
```

Comment:
Can you also remove the backticks here? In general: no backticks in the type descriptions